src/main/java/games/strategy/engine/framework/GameDataUtils.java

```diff
@@ -39,22 +38,23 @@ public static GameData cloneGameData(final GameData data, final boolean copyDele
   @SuppressWarnings("unchecked")
   public static <T> T translateIntoOtherGameData(final T object, final GameData translateInto) {
     try {
-      ByteArrayOutputStream sink = new ByteArrayOutputStream(1024);
-      try (final GameObjectOutputStream out = new GameObjectOutputStream(sink)) {
+      try (final ByteArrayOutputStream sink = new ByteArrayOutputStream(1024);
```

Comment:
Would you mind merging this try-with-resource block with the outer try statement?